modules/tensor_mechanics/include/materials/ADComputeElasticityTensorBase.h

```diff
@@ -0,0 +1,58 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef ADCOMPUTEELASTICITYTENSORBASE_H
+#define ADCOMPUTEELASTICITYTENSORBASE_H
+
+#include "ADMaterial.h"
+#include "RankFourTensor.h"
+#include "GuaranteeProvider.h"
+#include "DerivativeMaterialPropertyNameInterface.h"
+
+#define usingComputeElasticityTensorBaseMembers                                                    \
+  usingMaterialMembers;                                                                            \
+  using ADComputeElasticityTensorBase<compute_stage>::_elasticity_tensor_name;                     \
+  using ADComputeElasticityTensorBase<compute_stage>::_elasticity_tensor;                          \
+  using ADComputeElasticityTensorBase<compute_stage>::issueGuarantee;
```

Comment:
The reason why the compiler yelled at you for having two `;` is because you finish this macro with a `;`. To keep with our current design pattern you should remove the `;` here and add back the `;` when invoking the macro

Reply:
Oh wow! I would have never caught that...